src/AnimationWrapper.tsx

```diff
@@ -1,5 +1,6 @@
 "use client";
```

Comment:
I think this can be removed, since it will be added from `tsup` via `esbuildOptions` property 🤔